nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LookupRecord.java

```diff
@@ -536,6 +536,8 @@ public int getLookupCount() {
     private class RecordPathReplacementStrategy implements ReplacementStrategy {
         private int lookupCount = 0;
 
+        private ArrayList SCHEMA_LOOKUP_RESULT_CACHE = new ArrayList();
```

Comment:
Although this isn't a CheckStyle error, we usually reserve variable names in all caps to constants. Perhaps change to `schemaLookupResultCache`?